volttron/platform/web/master_web_service.py

```diff
@@ -337,7 +338,15 @@ def _get_discovery(self, environ, start_response, data=None):
 
         return_dict['vip-address'] = external_vip
         if self.core.messagebus == 'rmq':
-            return_dict['vc-rmq-address'] = self.volttron_central_rmq_address
+            config = RMQConfig()
+            rmq_address = None
+            if config.is_ssl:
```

Comment:
seems like you can just change the protocol in the if statement and add it as another parameter rather than the two lines that are exactly the same but the s in amqps

Reply:
Port also is changing with amqp/s.

Reply:
Ahh ok